.drone.yml

```diff
@@ -25,8 +25,6 @@ platform:
   arch: amd64
 
 trigger:
-# branch:
-#   - main
```

Comment:
You'll presumably want this only to trigger on your `cms-data-migration` branch, not every branch ever.